test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,21 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @test
+    public void isAnyNull() throws Exception {
```

Comment:
This is a correctly written test. Can you think of splitting them to multiple tests with 3 part name format?